docs/documentation.njk

```diff
@@ -153,7 +157,7 @@ <h3 id="generating-a-parser-command-line">Command Line</h3>
 <pre><code class="language-console">$ peggy arithmetics.pegjs</code></pre>
 
 <p>This writes parser source code into a file with the same name as the grammar
-file but with “.js” extension. You can also specify the output file
+file but with &ldquo;.js&rdquo; extension. You can also specify the output file
```

Comment:
These quotes were causing issues with the template parsing, so I converted them to HTML escape characters, which seemed to work.

Reply:
Oh, one other thing, I'd like the examples in documentation.html to be syntax-highlighted one day.  I mention this because in the GitHub Markdown, `peg.js` is a supported language.